packages/postgres-database/src/simcore_postgres_database/models/projects_metadata.py

```diff
@@ -47,10 +48,29 @@
         server_default=sa.text("'{}'::jsonb"),
         doc="Reserved for the user to store custom metadata",
     ),
+    sa.Column(
+        "parent_project_uuid",
+        sa.String,
+        nullable=True,
+        doc="If applicable the parent project UUID of this project",
+    ),
+    sa.Column(
+        "parent_node_id",
+        sa.String,
+        nullable=True,
+        doc="If applicable the parent node UUID of this project",
```

Comment:
same here

Reply:
@GitHK let's discuss on Thursday. I don't get what you don't get ;)